app/lib/features/chat_ng/widgets/events/text_message_event.dart

```diff
@@ -90,7 +91,7 @@ class TextMessageEvent extends StatelessWidget {
     final textTheme = Theme.of(context).textTheme;
     final chatTheme = Theme.of(context).chatTheme;
     final colorScheme = Theme.of(context).colorScheme;
-    final body = content.formattedBody() ?? content.body();
+    final body = content.formattedBody() ?? md.markdownToHtml(content.body());
```

Comment:
had to bring markdown utility to convert to html as ```flutter_matrix_html``` doesn't support markdown format.